sdk/src/wallet/operations/transaction/prepare_transaction.rs

```diff
@@ -6,8 +6,6 @@ use std::collections::HashSet;
 use instant::Instant;
 use packable::bounded::TryIntoBoundedU16Error;
 
-#[cfg(feature = "events")]
-use crate::wallet::events::types::{AddressData, TransactionProgressEvent, WalletEvent};
```

Comment:
I was wondering why these were warnings in my PR. Are we not triggering an event anymore in this file?

Reply:
No, but also didn't check now what it was before and if it's somewhere else or not

Reply:
In my PR I've put a TODO because this seems super suspicious to me. Like a failed merge or something.

Reply:
Checked now, it was for the case when we generate a remainder address, which doesn't happen anymore
https://github.com/iotaledger/iota-sdk/blob/bd7c08f8b9a065312058a2279e161e8193f2a35a/sdk/src/wallet/account/operations/transaction/prepare_transaction.rs#L88C63-L88C96

Reply:
Thanks